src/vcState.h

```diff
@@ -115,6 +115,8 @@ struct vcState
   vcLoginStatus loginStatus;
   vdkError logoutReason;
 
+  vdkError lastError;
```

Comment:
This isn't a good system; having a single value for an error isn't enough- you should leverage the file error system or modify that to suit instead.